packages/core/src/components/meter/meter.scss

```
@@ -0,0 +1,86 @@
+:host {
+  align-items: flex-start;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  width: 100%;
+  word-wrap: break-word;
+}
+
+.container-text {
+  align-items: flex-end;
+  display: flex;
+  font: var(--text-body-small);
+  font-weight: 300;
+  justify-content: space-between;
+  letter-spacing: var(--text-body-small-letter);
+  margin-bottom: var(--spacing-xsmall);
+  width: 100%;
+
+  &.hasCenterTitle {
```

block: out of our patterns

```suggestion
  &.has-center-title {
```

---

Nice!

Made: https://github.com/juntossomosmais/atomium/pull/695/commits/baaf5db51375ee72c2721e63925bb80bc036bb89